mesheryctl/internal/cli/root/mesh/validate.go

```diff
@@ -24,84 +21,65 @@ type Operation struct {
 	Query      string `json:"query"`
 }
 
-var spec string
-var adapterURL string
-var namespace string
-var watch bool
-var err error
-
 // validateCmd represents the service mesh validation command
-var validateCmd = &cobra.Command{
-	Use:   "validate",
-	Short: "Validate conformance to service mesh standards",
-	Args:  cobra.NoArgs,
-	Example: `
-// Validate conformance to service mesh standards
-mesheryctl mesh validate --adapter [name of the adapter] --tokenPath [path to token for authentication] --spec [specification to be used for conformance test] --namespace [namespace to be used]
-
-! Refer below image link for usage
-* Usage of mesheryctl mesh validate
-# ![mesh-validate-usage](/assets/img/mesheryctl/mesh-validate.png)
```

Comment:
We might need examples here (restore might be needed)! Else the CI action won't update the docs, not big issue here but conveying

Reply:
I can add examples. I would prefer not to include this part though:
```
! Refer below image link for usage
* Usage of mesheryctl mesh validate
# ![mesh-validate-usage](/assets/img/mesheryctl/mesh-validate.png)
```
It doesn't make sense to have users open an image to see the usage when cobra can display it right here.